server/service/appconfig.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/kolide/kit/version"
 )
 
+const defaultTransparencyURL = "https://fleetdm.com/transparency"
```

Comment:
We could define and export this in `server/fleet` (it's been copy/pasted in many places).